controllers/artefacts.rb

```diff
@@ -183,8 +193,51 @@ def get_ontology_and_latest_submission
                 @latest_submission.bring(ontology: [:acronym])
                 return @ontology, @latest_submission
             end
+            
+            def fetch_resource
```

Comment:
we have do something similar for the derefencement logic in the UI and OLD, 
UI: https://github.com/ontoportal-lirmm/bioportal_web_ui/blob/master/app/controllers/concerns/uri_redirection.rb#L51
OLD: https://github.com/ontoportal-lirmm/ontologies_linked_data/blob/master/lib/ontologies_linked_data/models/resource.rb#L141

Maybe think of how we harmonize all of this if possible.
Another remarks it to more this function to the search helper.

Reply:
yes good idea to harmonize and make all this linked together. i'll see how we can make these all linked.